public/javascripts/SVLabel/src/SVLabel/keyboard/Keyboard.js

```diff
@@ -61,8 +61,18 @@ function Keyboard (svl, canvas, contextMenu, googleMap, ribbon, zoomControl) {
      */
     this._documentKeyDown = function (e) {
         // The callback method that is triggered with a keyUp event.
-        if (contextMenu.isOpen()) {
-            return;
+        if (!status.focusOnTextField) {
+            if (contextMenu.isOpen()) {
+                if (e.keyCode == 16) {
+                    status.shiftDown = true;
+                }//only permits shift key value change when ctxt menu open
+                //kept separate from switch statement in elseif
+                if (e.keyCode == 187 || e.keyCode == 189) {
+                    svl.contextMenu.hide();
+                }
+                //equal button || - button
```

Comment:
Also, could you update the comments to be placed above the corresponding code here and in the future? It would be good for consistency since all the comments are done in this way in the codebase.